src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
you don't need `health` passed over constructor, this one is set by default under the hood
```suggestion
  constructor(name) {
    this.health = 100;
    this.name = name;
```